edocumentation-domain/src/main/java/org/dcsa/edocumentation/domain/persistence/entity/ActiveReeferSettings.java

```diff
@@ -26,58 +26,73 @@ public class ActiveReeferSettings {
   @Column(name = "id", nullable = false)
   private UUID id;
 
+  @Enumerated(EnumType.STRING)
   @Column(name = "reefer_type_code", nullable = false)
-  private String type;
+  private ReeferType type;
 
+  @Builder.Default
   @Column(name = "is_cargo_probe_1_required", nullable = false)
-  private Boolean isCargoProbe1Required;
+  private Boolean isCargoProbe1Required = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_cargo_probe_2_required", nullable = false)
-  private Boolean isCargoProbe2Required;
+  private Boolean isCargoProbe2Required = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_cargo_probe_3_required", nullable = false)
-  private Boolean isCargoProbe3Required;
+  private Boolean isCargoProbe3Required = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_cargo_probe_4_required", nullable = false)
-  private Boolean isCargoProbe4Required;
+  private Boolean isCargoProbe4Required = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_ventilation_open", nullable = false)
-  private Boolean isVentilationOpen;
+  private Boolean isVentilationOpen = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_drainholes_open", nullable = false)
-  private Boolean isDrainholesOpen;
+  private Boolean isDrainholesOpen = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_bulb_mode", nullable = false)
-  private Boolean isBulbMode;
+  private Boolean isBulbMode = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_gen_set_required", nullable = false)
-  private Boolean isGenSetRequired;
+  private Boolean isGeneratorSetRequired = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_pre_cooling_required", nullable = false)
-  private Boolean isPreCoolingRequired;
+  private Boolean isPreCoolingRequired = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_cold_treatment_required", nullable = false)
-  private Boolean isColdTreatmentRequired;
+  private Boolean isColdTreatmentRequired = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_hot_stuffing_allowed", nullable = false)
-  private Boolean isHotStuffingAllowed;
+  private Boolean isHotStuffingAllowed = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_tracing_required", nullable = false)
-  private Boolean isTracingRequired;
+  private Boolean isTracingRequired = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_monitoring_required", nullable = false)
-  private Boolean isMonitoringRequired;
+  private Boolean isMonitoringRequired = Boolean.FALSE;
 
+  @Builder.Default
   @Column(name = "is_high_value_cargo", nullable = false)
-  private Boolean isHighValueCargo;
+  private Boolean isHighValueCargo = Boolean.FALSE;
 
   @Column(name = "product_name", length = 500)
   private String productName;
 
   @Column(name = "extra_material", length = 500)
   private String extraMaterial;
 
-  @ToString.Exclude
+  // @ToString.Exclude
```

Comment:
Please delete the line if it is not needed anymore

Reply:
Should probably still there. Just needed it temporarily for debug reasons